corehq/apps/ota/views.py

```diff
@@ -265,11 +265,20 @@ def get_restore_response(domain, couch_user, app_id=None, since=None, version='1
     app = get_app_cached(domain, app_id) if app_id else None
 
     #For ICDS only allow users who have access the specific app
```

Comment:
The containing method is already pretty bloated, can we try and extract this out?
Can also move the extraction to `custom/icds/view_utils`.

Reply:
I was thinking about that too, will do it once things are finalized.